packages/opentelemetry-node/docs/metrics.asciidoc

```diff
@@ -1,30 +1,49 @@
----
-id: otelNodeMetrics
-slug: /otel-nodejs/metrics
-title: Metrics
----
+////
+Goal of this doc:
+???
 
-## Enabled by default
+Assumptions we're comfortable making about the reader:
+- ???
+- ???
+- ???
+////
+
+[[metrics]]
+== Metrics
+
+:language: Node.js
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+[discrete]
+[[metrics-enabled-by-default]]
+=== Enabled by default
 
 In this distribution metrics are enabled by default and sent to the endpoint
 configured by you. If you wish to disable metrics you can by setting the env
 variable `ELASTIC_OTEL_METRICS_DISABLED` to the string `true`.
 
-```sh
+[source,sh]
+----
 export ELASTIC_APM_SERVER_URL="https://apm-server-host.co"
 export ELASTIC_APM_SECRET_TOKEN="secret_token"
```

Comment:
```suggestion
```

These ELASTIC_ envvars are copy pasta from our classic non-OTel APM agent.  Clearly we (devs) need to review this doc too. :)